src/RecordTask.h

```diff
@@ -584,7 +584,7 @@ class RecordTask final : public Task {
   bool is_container_init() const { return tg->tgid_own_namespace == 1; }
 
   /**
-   * Linux requires the invariant that that all members of a thread group
```

Comment:
sometimes doubled `that` is intentional. It didn't appear to be in this repository